frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -92,13 +85,10 @@ class Worksheet extends React.Component {
             BulkBundleDialog: null,
             showBundleOperationButtons: false,
             uuidBundlesCheckedCount: {},
-            openDelete: false,
-            openDetach: false,
-            openKill: false,
-            openDeleteItem: false,
+            openDialog: null,
```

Comment:
Let's rename this to `openedDialog` -- that way, it's clear that this is a variable, as opposed to `closeDialog`, which is a function.